src/asm/charmap.c

```diff
@@ -44,10 +44,10 @@ struct Charmap {
 
 static HashMap charmaps;
 
-static struct Charmap *currentCharmap;
+static struct Charmap **currentCharmap;
```

Comment:
I would like an additional comment explaining why we're using a double pointer instead of a simple pointer.